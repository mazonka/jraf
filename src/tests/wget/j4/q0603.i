command=jw md 0 /.jraf.sys/users : login admin@server.com * : read f82d3bb17612458c /.jraf.sys/login : logout f82d3bb17612458c : read f82d3bb17612458c /.jraf.sys/login