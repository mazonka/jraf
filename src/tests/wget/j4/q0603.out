OK /.jraf.sys/users : OK http://127.0.0.1:16000/jraf? : OK 0 -1 1 0 16 f82d3bb17612458c : OK : AUTH
==
34d3ae7c5e2cdebb
